docs/architecture/grpc-for-wcf-developers/call-credentials.md

```diff
@@ -6,27 +6,25 @@ ms.date: 09/02/2019
 
 # Call credentials
 
-Call credentials are all based on some kind of token passed in metadata with each request.
+Call credentials are all based on a token passed in metadata with each request.
 
 ## WS-Federation
 
-ASP.NET Core supports WS-Federation using the [WsFederation](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.WsFederation) NuGet package. WS-Federation is the closest available alternative to Windows Authentication, which is not supported over HTTP/2. Users are authenticated using Active Directory Federation Services (ADFS), which provides a token that can be used to authenticate with ASP.NET Core.
+ASP.NET Core supports WS-Federation using the [WsFederation](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.WsFederation) NuGet package. WS-Federation is the closest available alternative to Windows Authentication, which isn't supported over HTTP/2. Users authenticate by using Active Directory Federation Services (AD FS), which provides a token to authenticate with ASP.NET Core.
```

Comment:
```suggestion
ASP.NET Core supports WS-Federation using the [WsFederation](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.WsFederation) NuGet package. WS-Federation is the closest available alternative to Windows Authentication, which isn't supported over HTTP/2. Users authenticate by using Active Directory Federation Services (AD FS), which provides a token that can be used to authenticate with ASP.NET Core.
```

I feel like this changes the meaning. AD FS doesn't directly authenticate with ASP.NET Core so it's important to make that distinction that you're only issued a token.